htsinfer/get_library_source.py

```diff
@@ -270,6 +271,11 @@ def get_source_expression(
                 "Could not process file: {_file}"
             ) from exc
 
+        if dat.empty:
+            raise ValueError(
```

Comment:
What is the possible cause and consequence of this? No chance of recovering from this (e.g., by setting the library source to `UNKNOWN` or whatever the appropriate state is)? Not quite sure whether raising an error is the best choice here.

If it _does_ turn out that raising an error is the best approach, then please list the exception and its cause in the docstring under `FileProblem`.